cli/src/cluster_query.rs

```diff
@@ -801,6 +817,7 @@ pub fn process_catchup(
         sleep(Duration::from_secs(sleep_interval as u64));
```

Comment:
let's stick `interval_count += 1;` to this sleep because these are more tightly bound together than `previous_rpc_slot`/`previous_slot_distance`.

Reply:
Also, this is personal opinion, but I think `total_sleep_interval += sleep_interval` is simpler and you don't need `interval_count * sleep_interval` above. Simpler code, less bugs. :)

You don't need to care about this usually, but in that way, we reduce calculations from `+=` and `*` to only `+=` per this loop iteration.